src/jq/api.clj

```diff
@@ -33,9 +33,9 @@
   ([^String query opts]
    (let [^JsonQuery json-query (impl/compile-query query)
          ^Scope scope (impl/new-scope opts)]
-     (fn ^String [^String data]
+     (fn ^String [^String data & {:keys [vars]}]
```

Comment:
to keep it consistent I'd prefer to make this function with two arguments, first for data, second for options map.

Reply:
Updated accordingly in 3dcaedfd5e5fd5bda78008ee7767ebb45028a9f4

Reply:
(that committish is no longer in history as a separate item, but its contents have been folded in when rebasing/squashing this PR onto current main).